lib/event/member_joined_channel.rb

```diff
@@ -0,0 +1,42 @@
+require 'operationcode/slack'
+
+# Handler for a Member Joined Channel event. When a user joins a Slack channel, we want to:
+# * Check what channel is joined, and take additional actions on a per- channel basis as needed
+
+class Event
```

Comment:
Instead of this being a class nested in a class, we can use the module name spacing syntax, like in [team_join.rb](https://github.com/OperationCode/operationcode_bot/blob/master/lib/event/team_join.rb#L9).

So lines 6 and 7 would be collapsed into `class Event::MemberJoinedChannel < Event`